virttest/libvirt_xml/nodedev_xml.py

```diff
@@ -91,17 +91,58 @@ def get_sysfs_sub_path(self):
 
 
 class NetXML(CAPXML):
-
     """
     class for capability whose type is net.
     """
-    __slots__ = ('interface', 'address')
+    # Example:
+    #<capability type='net'>
+    #<interface>eno1</interface>
+    #<address>44:37:e6:95:03:e4</address>
+    #<link speed='100' state='up'/>
+    #<feature name='rx'/>
+    #<feature name='tx'/>
+    #<capability type='80203'/>
+    #</capability>
+    #</device>
+
+    __slots__ = ('address', 'link_speed', 'link_state')
```

Comment:
we may keep internet attribute as it before

Reply:
done , add the 'interface' back